cypress/features/regression/multiSelect.feature

```diff
@@ -143,8 +143,8 @@ Feature: Multi Select component
   @positive
   Scenario: Check the onKeyDown event after clicking arrow
     Given I open "Select multiselect" component page "default story"
-    When I focus default Select input
-      And I click onto default select using "downarrow" key
+    When I click on controlled Select input
+      And I click onto controlled select using "downarrow" key
```

Comment:
As above, regarding "click' with a downarrow key.

Reply:
how would you prefer to name it ?

Reply:
How about `And I press "downarrow" key on controlled select` as you don't click the downarrow key, you press it.